src/ILCompiler.ManagedAssembliesBuildTasks/src/ILCompiler.ManagedAssembliesBuildTask.csproj

```diff
@@ -0,0 +1,25 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" Condition="'$(IsProjectNLibrary)' != 'true'" />
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RootNamespace>ILCompiler</RootNamespace>
+    <AssemblyName>ILCompiler.ManagedAssembliesBuildTask</AssemblyName>
```

Comment:
I would give it a bit more generic name, e.g. `ILCompiler.Build.Tasks`. It is likely we will come up with more assorted build tasks to add to this assembly over time.